extensions/wikia/CreateNewWiki/CreateWiki.php

```diff
@@ -1189,4 +1189,13 @@ public function getWikiInfo($key) {
 		$ret = $this->mNewWiki->$key;
 		return $ret;
 	}
+
+	/**
+	 * sets initial value for wgEnableNjordExt for new created wiki
+	 * TODO: for first phase of prototype set to true for 10% of new created english wikis only
+	 * @return bool
+	 */
+	private function setInitialNjordExtValue() {
+		return intval( $this->mNewWiki->city_id ) % 10 === 1 && $this->mNewWiki->language === 'en' ? true : false;
```

Comment:
In this pull request only wikis with city_id ending with 1 that are in EN will have Hero Image enabled.
You might want to consider using rand(0,9) instead of $this->mNewWiki->city_id to ensure fair treatment of all IDs.